adal/util.py

```diff
@@ -55,7 +54,7 @@ def add_default_request_headers(self, options):
     headers[AdalIdParameters.SKU] = AdalIdParameters.PYTHON_SKU
     headers[AdalIdParameters.VERSION] = adal.__version__
     headers[AdalIdParameters.OS] = sys.platform
-    headers[AdalIdParameters.CPU] = 'x64' if platform.architecture()[0] == '64bit' else 'x86'
+    headers[AdalIdParameters.CPU] = 'x64' if sys.maxsize > 2 ** 32 else 'x86'

```

Comment:
if we mean to get the maximum possible interger, then I guess we should compare with '2**31 -1 '
```
>>> sys.maxsize
2147483647
>>> hex(sys.maxsize)
'0x7fffffff'
```